src/cgame/cg_gameinfo.cpp

```diff
@@ -41,11 +41,11 @@ CG_ParseInfos
 int CG_ParseInfos( const char *buf, int max, char *infos[] )
 {
 	char *token;
-	int  count;
+	int  count = 0;
 	char key[ MAX_TOKEN_CHARS ];
 	char info[ MAX_INFO_STRING ];
 
-	count = 0;
+	int infoPostfixLen = strlen( "\\num\\" ) + strlen( va( "%d", MAX_ARENAS ) );
```

Comment:
In that case you could just use malloc.
And there might be parts of the code that use whatever is being returned from BG_Alloc, either directly or indirectly, that expect the whole buffer to be zero-initialized.
So I'd just leave it as it is, it'd be very easy to introduce silent bugs that are extremely hard to spot.